### PR TITLE
Bug 1129344 - Do not focus search window on opened +autoland

### PR DESCRIPTION
**Pull request by alivedise**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f01d2c2a8226)

File: apps/system/test/unit/app_transition_controller_test.js [hunk 1]
```diff
@@ -279,6 +279,21 @@ suite('system/AppTransitionController', function() {
     assert.isTrue(stubFocus.notCalled);
   });
 
+  test('Do not focus if we are search window', function() {
```
Comment (etiennesegonzac) [inline apps/system/test/unit/app_transition_controller_test.js]: nit: `Do not focus search windows`

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9856dee9a488)

**Comment by try-server-hook** [comment]
[alivedise Alive.Kuo (alivedise)](https://github.com/alivedise) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c78fca6149e5)

**Pull request merged**